util/config_stripper.py

```diff
@@ -78,11 +95,61 @@ def strip_tar(input, output):
     shutil.rmtree(tempdir)
     return 0
 
-
-def strip_config(path):
+def strip_layer(path):
+    # The original layer tar is of the form <random string>/layer.tar, the
+    # working directory is one level up from where layer.tar is.
+    original_dir = os.path.normpath(os.path.join(os.path.dirname(path), '..'))
+
+    buf = cStringIO.StringIO()
+
+    # Go through each file/dir in the layer
+    # Set its mtime to 0
+    # If it's a file, add its content to the running buffer
+    # Add it to the new gzip'd tar.
+    with tarfile.open(name=path, mode='r') as it:
+      with tarfile.open(fileobj=buf, mode='w') as ot:
+        ti = it.next()
```

Comment:
I think you can do "for file in it" or something like that to avoid the while loop

Reply:
Yep. Fixed.